Runs: Rewrite Runs section (#2011)
wandb/docs: models/runs/view-logged-runs.mdx

---
title: View a specific run in a project
description: Learn how to view a specific logged run and its properties using the W&B App or the LEET terminal UI.
---

> View information about a specific run such as the state of the run, artifacts logged to the run, metrics logged to a run, system metrics, and more.

Reviewer: ```suggestion
View information about a specific run, such as its current state, artifacts, metrics, and more.
```

## View a logged run

You can view and monitor runs using the W&B App or the `wandb beta leet` terminal UI.

<Tabs>
<Tab title="WA&B App">

To view a specific run in the W&B App:

1. Navigate to the [W&B App](https://wandb.ai/home).
2. Navigate to the W&B project you specified when you initialized the run.
3. Within the project sidebar, select the **Workspace** tab.
4. Within the run selector, click the run you want to view, or enter a partial run name to filter for matching runs.

Alternatively, you can directly access a specific run's workspace by entering its URL in your browser. The URL path of a specific run has the following format:

```text
https://wandb.ai/<team-name>/<project-name>/runs/<run-id>
```

> Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and [run ID](/models/runs/run-identifiers#run-id).

Reviewer: ```suggestion
Replace values enclosed in angle brackets (`< >`) with the actual values of the team name, project name, and [run ID](/models/runs/run-identifiers#run-id).
```

> After you select a run, explore its properties by navigating through the tabs: [Overview](/models/runs/view-logged-runs#overview), [Logs](/models/runs/view-logged-runs#logs), [Files](/models/runs/view-logged-runs#files), [Code](/models/runs/view-logged-runs#code), and [Artifacts](/models/runs/view-logged-runs#artifacts).

Reviewer: ```suggestion
Explore the run's properties by navigating through the tabs: [Overview](/models/runs/view-logged-runs#overview), [Logs](/models/runs/view-logged-runs#logs), [Files](/models/runs/view-logged-runs#files), [Code](/models/runs/view-logged-runs#code), and [Artifacts](/models/runs/view-logged-runs#artifacts).
```

2. Start `wandb beta leet` using one of these commands:

   ```bash
   # View the latest run, stored in ./wandb/latest-run/
   wandb beta leet
   
   # Specify a run directory
   wandb beta leet ./wandb/run-20250813_124246-n67z9ude
   
   # Specify a .wandb file
   wandb beta leet ./wandb/run-20250813_124246-n67z9ude/run-n67z9ude.wandb
   ```

![Screenshot demonstrating viewing a run in the LEET terminal UI](/images/leet/leet-tui.png)

LEET displays a three-panel interface:

- **Left sidebar**: Run overview with environment variables, configuration, and summary statistics
- **Center panel**: Metrics grid with Braille-style line charts showing your logged metrics
- **Right sidebar**: System metrics including GPU/CPU/RAM utilization

Get started with these keyboard shortcuts:

- `h` or `?` - View all keyboard shortcuts
- `/` - Filter metrics by pattern
- `[` / `]` - Toggle left/right sidebars
- `n` / `N` - Navigate between metric pages
- `q` / `CMD+C` - Quit

See the [`wandb beta leet`](/models/ref/cli/wandb-beta/wandb-beta-leet) reference for more details.

</Tab>
</Tabs>

> ### Overview

Reviewer: I'm not sure but I think that all of these H3s relate only to the W&B App tab above, not the LEET tab. Can you confirm?

Author: Confirming. It's odd. Maybe I should split the tabs and create new sections (W&B App and LEET, respectively). Then nest the current H3s into that? 🤔
